### ISSUE TITLE
Page.derotation_matrix returns a tuple instead of a Matrix with rebased implementation

### ISSUE THREAD

**Issue by jn-chrn** [pymupdf/PyMuPDF#2911]
### Description of the bug

With PyMuPDF latest version (1.23.8) and the rebased implementation (and maybe also before), the derotation matrix of a page, which can be accessed with the property `Page.derotation_matrix` returns a tuple. The documentation states that a matrix should be returned. 

For the rotation matrix, a `fitz_new.Matrix` is returned, which matches the documentation. 

### How to reproduce the bug

The following lines of code show the type of the rotation and derotation matrices for an empty document:

```python
import fitz_new as fitz

doc = fitz.open()
page = doc.new_page()
print(f"{type(page.rotation_matrix)=}")
print(f"{type(page.derotation_matrix)=}")
print(f"{page.derotation_matrix=}")
```

Output:

```
type(page.rotation_matrix)=<class 'fitz_new.Matrix'>
type(page.derotation_matrix)=<class 'tuple'>  # <--- should be a fitz_new.Matrix as well
page.derotation_matrix=(1.0, -0.0, -0.0, 1.0, 0.0, 0.0)
```

In the output, one who expect both matrices to have the type `fitz_new.Matrix`.

### PyMuPDF version

1.23.8

### Operating system

Linux

### Python version

3.11

**Comment by julian-smith-artifex-com** [comment]
Thanks for reporting this. As you can see i have just created a PR with a fix.

**Comment by jn-chrn** [comment]
Thanks for the quick fix as usual!

**Comment by julian-smith-artifex-com** [comment]
This is fixed in release candidate 1.23.9rc1; see: #2960

**Comment by julian-smith-artifex-com** [comment]
Fixed in 1.23.9.
